pkg/controllers/user/monitoring/projectHandler.go

```diff
@@ -167,32 +175,18 @@ func (ph *projectHandler) deployApp(appName, appTargetNamespace string, appProje
 
 		"exporter-fluentd.enabled": "false",
 
-		"grafana.enabled":            "true",
-		"grafana.level":              "project",
-		"grafana.apiGroup":           monitoring.APIVersion.Group,
-		"grafana.serviceAccountName": appName,
-
-		"prometheus.enabled":                          "true",
-		"prometheus.externalLabels.prometheus_from":   clusterName,
-		"prometheus.level":                            "project",
-		"prometheus.apiGroup":                         monitoring.APIVersion.Group,
-		"prometheus.serviceAccountNameOverride":       appName,
-		"prometheus.additionalBindingClusterRoles[0]": fmt.Sprintf("%s-namespaces-readonly", appDeployProjectID),
-		"prometheus.additionalAlertManagerConfigs[0].static_configs[0].targets[0]":          fmt.Sprintf("%s.%s:%s", clusterAlertManagerSvcName, clusterAlertManagerSvcNamespaces, clusterAlertManagerPort),
-		"prometheus.additionalAlertManagerConfigs[0].static_configs[0].labels.level":        "project",
-		"prometheus.additionalAlertManagerConfigs[0].static_configs[0].labels.project_id":   project.Name,
-		"prometheus.additionalAlertManagerConfigs[0].static_configs[0].labels.project_name": project.Spec.DisplayName,
-		"prometheus.additionalAlertManagerConfigs[0].static_configs[0].labels.cluster_id":   project.Spec.ClusterName,
-		"prometheus.additionalAlertManagerConfigs[0].static_configs[0].labels.cluster_name": clusterName,
-		"prometheus.serviceMonitorNamespaceSelector.matchExpressions[0].key":                nslabels.ProjectIDFieldLabel,
-		"prometheus.serviceMonitorNamespaceSelector.matchExpressions[0].operator":           "In",
-		"prometheus.serviceMonitorNamespaceSelector.matchExpressions[0].values[0]":          appDeployProjectID,
-		"prometheus.ruleNamespaceSelector.matchExpressions[0].key":                          nslabels.ProjectIDFieldLabel,
-		"prometheus.ruleNamespaceSelector.matchExpressions[0].operator":                     "In",
-		"prometheus.ruleNamespaceSelector.matchExpressions[0].values[0]":                    appDeployProjectID,
-		"prometheus.ruleSelector.matchExpressions[0].key":                                   monitoring.CattlePrometheusRuleLabelKey,
-		"prometheus.ruleSelector.matchExpressions[0].operator":                              "In",
-		"prometheus.ruleSelector.matchExpressions[0].values[0]":                             monitoring.CattleAlertingPrometheusRuleLabelValue,
+		"grafana.enabled":  "true",
+		"grafana.level":    "project",
+		"grafana.apiGroup": monitoring.APIVersion.Group,
+
+		"prometheus.enabled":                       "true",
+		"prometheus.level":                         "project",
+		"prometheus.apiGroup":                      monitoring.APIVersion.Group,
```

Comment:
Just curious why `apiGroup` is a configurable option of the chart

Reply:
That's a legacy problem. The initial design didn't use `monitoring.coreos.com` api group so we need it configurable.